Controller/Admin/Transaction/TransactionTabPostProcessing.php

```diff
@@ -109,32 +158,62 @@ private function _getRequestParameters(): array
      * @param array $aRequestParameters
      * @throws Exception
      *
-     * @since 1.0.0
+     * @since 1.0.1
      */
     private function _validateRequest(array $aRequestParameters)
     {
-        $fAmount = $aRequestParameters['amount'];
+        $fAmount = $aRequestParameters[self::KEY_AMOUNT];
 
-        if (!$fAmount || !is_numeric($fAmount)) {
+        if (!$this->_isAmountNumeric($fAmount)) {
             throw new Exception(Helper::translate('text_generic_error'));
         }
 
-        if ($fAmount < 0 || $fAmount > $this->oTransaction->wdoxidee_ordertransactions__amount->value) {
+        if (!$this->_isInRange($fAmount, 0, $this->oTransaction->wdoxidee_ordertransactions__amount->value)) {
             throw new Exception(Helper::translate('total_amount_not_in_range_text'));
         }
     }
 
+    /**
+     * Checks that the amount argument is a numeric value
+     *
+     * @param float $fAmount
+     *
+     * @return boolean true if $fAmount is a numeric value
+     *
+     * @since 1.0.1
+     */
+    private function _isAmountNumeric($fAmount)
+    {
+        return $fAmount && (is_float($fAmount) || is_int($fAmount));
+    }
+
+    /**
+     * Checks that the amount is in the range of the transaction
+     *
+     * @param float $fAmount
+     * @param float $fMinAmount (excluding this value)
```

Comment:
I think it should be either both **excluding or including**, but not a mixture as it is quite counter-intuitive.